project/RELEASE-CHECKLIST.md

```diff
@@ -312,16 +312,17 @@ echo "Linux 64bit tgz: https://test.docker.com/builds/Linux/x86_64/docker-${VERS
 echo "Windows 64bit client binary: https://test.docker.com/builds/Windows/x86_64/docker-${VERSION#v}.exe"
 echo "Windows 32bit client binary: https://test.docker.com/builds/Windows/i386/docker-${VERSION#v}.exe"
 ```
+### 13. Announce the release candidate
 
-We recommend announcing the release candidate on:
+The release candidate should be announced on:
 
 - IRC on #docker, #docker-dev, #docker-maintainers
 - In a comment on the pull request to notify subscribed people on GitHub
 - The [docker-dev](https://groups.google.com/forum/#!forum/docker-dev) group
 - The [docker-maintainers](https://groups.google.com/a/dockerproject.org/forum/#!forum/maintainers) group
-- Any social media that can bring some attention to the release candidate
+- (Optional) Any social media that can bring some attention to the release candidate
 
```

Comment:
Yes, we should add slack here as well; let me do a follow up, because slack needs to be added to other places as well